.eslintrc.js

```diff
@@ -23,8 +23,7 @@ module.exports = {
         "eslint-plugin/report-message-format": ["error", "[^a-z].*\\.$"],
         "eslint-plugin/test-case-property-ordering": "error",
         "eslint-plugin/test-case-shorthand-strings": "error",
-        "rulesdir/multiline-comment-style": "error",
-        "rulesdir/no-useless-catch": "error"
```

Comment:
shouldn't the new core rule now be enabled for this repo?

Reply:
I thought that's covered by adding it to `default.yml`: https://github.com/eslint/eslint/pull/11198/files#diff-5d1f0612c1d7040125f0e4e9539c9eb8R121